packages/lesswrong/server/scripts/nullifyVotes.ts

```diff
@@ -1,12 +1,36 @@
 import Users from '../../lib/collections/users/collection';
-import { Vulcan } from '../vulcan-lib';
-import { nullifyVotesForUser } from '../callbacks';
+import { Globals } from '../vulcan-lib';
+import { nullifyVotesForUser, nullifyVotesForUserByTarget } from '../callbacks';
 
-Vulcan.nullifyVotesForNullifiedUsers = async () => {
+Globals.nullifyVotesForNullifiedUsers = async () => {
   const users = await Users.find({nullifyVotes: true}).fetch();
   users.forEach((user) => {
     void nullifyVotesForUser(user);
   })
   //eslint-disable-next-line no-console
   console.warn(`Nullified votes for ${users.length} users`);
 }
+
+interface DateRange {
```

Comment:
I think it's a a little confusing that these two files each declare an interface with the same name but incompatible types. Probably one should be renamed (`UserDateRange` or `DateRangeInput` maybe?), or the global function could just be `(sourceUserId: string, targetUserId: string, after?: string, before?: string)`

Reply:
Yeah, `DateRangeInput` seems good.